src/brs/http/APIServlet.java

```diff
@@ -37,6 +37,7 @@ public APIServlet(TransactionProcessor transactionProcessor, Blockchain blockcha
 
     enforcePost = propertyService.getBoolean(Props.API_SERVER_ENFORCE_POST);
     acceptSurplusParams = propertyService.getBoolean(Props.API_ACCEPT_SURPLUS_PARAMS);
+    allowedOrigins = propertyService.getString(Props.API_ALLOWED_ORIGINS);
```

Comment:
I wonder why Sonar is complaining this - looks good to me
![image](https://user-images.githubusercontent.com/3920663/54596921-62dd9180-4a14-11e9-858c-a83497688d9a.png)

Reply:
It's updating a static field in a constructor, which is really bad practice, but I really don't care. It's acceptable because the class is intended to be Singleton and the static field will always be assigned the same value. The rest of the class is written this way so keeping styling consistent... for now.